optuna/visualization/_slice.py

```diff
@@ -21,6 +24,82 @@
 _logger = get_logger(__name__)
 
 
+class _SliceSubplotInfo(NamedTuple):
+    param_name: str
+    x: List[Any]
+    y: List[float]
+    trial_numbers: List[int]
+    is_x_log_scale: bool
+    is_x_nunmerical: bool
+
+
+class _SlicePlotInfo(NamedTuple):
```

Comment:
It may be possible to have `_get_slice_plot_info` return a list of `_SliceSubplotInfo` without defining this class. In that case, it may be necessary to rename some classes and functions. (`_SliceSubplotInfo` -> `_SliceInfo`, `_get_slice_plot_info` -> `_get_slice_info_list`) What do you think?